.gitpod.yml

```
@@ -0,0 +1,14 @@
+image:
+  file: .gitpod.Dockerfile
```

Could this file be moved under a subfolder, even if `.gitpod.yml` is at the root?

---

I think the `gitpod.Dockerfile` must be in the root, at least that's what the docs seem to say: [^docs]

> ## Configure a custom Dockerfile
> 
> This option provides you with the most flexibility. Start by adding the following configuration in your .gitpod.yml file:
> 
> ```yaml
> image:
>   file: .gitpod.Dockerfile
> ```
> 
> Next, create a `.gitpod.Dockerfile` file at the root of your project. The syntax is the regular `Dockerfile` syntax as documented on docs.docker.com.

I have not tried if you can put the `.gitpod.Dockerfile` somewhere else and mention that in the `gitpod.yml` config...

[^docs]: https://www.gitpod.io/docs/config-docker#configure-a-custom-dockerfile

---

This comment implies it's possible: https://github.com/gitpod-io/gitpod/issues/7721#issuecomment-1018685624 (and maybe a different approach than this PR)

---

Please try to move to subdir.
```suggestion
  file: .gitpod/Dockerfile
```